packages/www/hooks/use-api.tsx

```diff
@@ -99,9 +99,10 @@ const makeContext = (state: ApiState, setState) => {
         headers.set("authorization", `JWT ${state.token}`);
       }
       const endpoint =
+        window.location.hostname.includes("livepeer.monster") ||
         window.location.hostname.includes("livepeerorg.vercel.app") ||
         window.location.hostname.includes("livepeerorg.now.sh")
-          ? `https://livepeer.monster/api${url}`
```

Comment:
This should get flipped back and we should make sure it works before shipping

Reply:
Done